modules/mono/mono_gd/gd_mono.cpp

```diff
@@ -690,6 +692,23 @@ bool GDMono::_load_api_assemblies() {
 	return true;
 }
 
+void GDMono::_install_trace_listener() {
+
+#ifdef DEBUG_ENABLED
+	// Install the trace listener now before the project assembly is loaded
+	typedef void (*DebuggingUtils_InstallTraceListener)(MonoObject **);
+	MonoException *exc = NULL;
+	GDMonoClass *debug_utils = core_api_assembly->get_class(BINDINGS_NAMESPACE, "DebuggingUtils");
+	DebuggingUtils_InstallTraceListener install_func =
+			(DebuggingUtils_InstallTraceListener)debug_utils->get_method_thunk("InstallTraceListener");
+	install_func((MonoObject **)&exc);
```

Comment:
It would be more readable without a method thunk. Something like this: `debug_utils->get_method("InstallTraceListener")->invoke(NULL, &exc)`.